shared/chat/conversation/messages/user-notice.js

```diff
@@ -16,7 +16,7 @@ const AVATAR_SIZE = 32
 
 const UserNotice = ({bgColor, username, teamname, children, style, onClickAvatar}: Props) => (
   <Box style={{...styleOuterBox, ...style}}>
-    {(username || teamname) && (
+    {!!(username || teamname) && (
```

Comment:
this would crash if someone did `<UserNotice teamname="">`, which we do in at least one place via `<UserNotice teamname={something || ''}>`